crypto-conditions/src/main/java/org/interledger/cryptoconditions/PreimageSha256Fulfillment.java

```diff
@@ -24,7 +24,7 @@
    */
   static PreimageSha256Fulfillment from(final byte[] preimage) {
     Objects.requireNonNull(preimage);
-    final String encodedPreimage = Base64.getUrlEncoder().encodeToString(preimage);
+    final String encodedPreimage = Base64.getEncoder().encodeToString(preimage);
 
```

Comment:
As discussed in #89 I think we should revert this.

If an external application needs the Base64 (not Bse64Url) encoded form it should convert between them

Reply:
Agree. Let's revert for now and discuss in #100.